pkg/provider-local/webhook/controlplane/ensurer.go

```diff
@@ -57,33 +84,53 @@ func (e *ensurer) EnsureAdditionalFiles(ctx context.Context, gc gcontext.GardenC
 		kindClusterName = "gardener-local-ha-control-plane"
 	}
 
+	var script bytes.Buffer
+	if err := tplInitializer.Execute(&script, map[string]interface{}{
+		"kindClusterName": kindClusterName,
+	}); err != nil {
+		return err
+	}
+
 	appendUniqueFile(new, extensionsv1alpha1.File{
-		Path:        "/etc/containerd/conf.d/50-provider-local-registry.toml",
```

Comment:
On existing nodes that already have this file, removing it from the OSC will not take effect. Hence the new script will only be active on new nodes.
I don't think this is problematic for provider-local, but should be kept in mind for upgrade tests etc.

Reply:
Thanks. Is there something to be done about it in this PR?

Reply:
Nope, just a heads-up when looking at the tests